src/main/scala/wdl4s/AstTools.scala

```diff
@@ -50,8 +50,8 @@ object AstTools {
               WdlArrayType(member)
             case "Map" =>
               if (subtypes.size != 2) throw new SyntaxError(wdlSyntaxErrorFormatter.mapMustHaveExactlyTwoTypeParameters(typeTerminal))
-              val keyType = subtypes(0).wdlType(wdlSyntaxErrorFormatter)
-              val valueType = subtypes(1).wdlType(wdlSyntaxErrorFormatter)
+              val keyType = subtypes.head.wdlType(wdlSyntaxErrorFormatter)
+              val valueType = subtypes.tail.head.wdlType(wdlSyntaxErrorFormatter)
```

Comment:
Twas our friend IntelliJ not liking the former line. The second one was me not liking one using `.head` and the other `(1)`.